README.md

```diff
@@ -171,12 +229,26 @@ horizontalpodautoscaler.autoscaling/puppetserver-compilers-autoscaler   Stateful
 
 ## Upgrading
 
-### from 7.x to 8.0
+### from Chart Version 7.x to 8.0
 
 `customPersistentVolumeClaim` was removed instead check `puppetdb.persistence.existingClaim` or `puppetserver.persistence.*.existingClaim` for similar functionnality.
 
 Added support for setting the size and Storage Class of individual Persistent Volume Claims fullfilling most uses of `customPersistentVolumeClaim` check `puppetdb.persistence.size` `puppetdb.persistence.storageClass`, `puppetserver.persistence.*.storageClass` and `puppetserver.persistence.*.size` for more information.
 
+### Puppet Version from 7.x to 8.x
```

Comment:
I don't think we should document that changing a container version tag allows you to change the version as that is a generic thing across all Helm charts and Docker in general. Or is there something more to this that I'm missing?

Reply:
RIght, i guess it makes sense to inform people that from the perspective of using the chart to uprade to Puppet 8 there is no additional steps required apart from changing the version. As you say, we should bump to v8 soon and release a major version of the chart.